mysql-test/main/max_used_connections_time.test

```diff
@@ -0,0 +1,35 @@
+-- source include/not_embedded.inc
+
+--echo #
+--echo # MDEV-30543 New status variable: Max_used_connections_time
+--echo #
+
+FLUSH STATUS;
```

Comment:
Do you need this at the beginning ?
Same for the next 2 lines?

Reply:
yes, other tests may have a higher number of max connections. Check `FLUSH STATUS` actually resets the value. I suspect it may not.